amundsen_application/static/js/components/DashboardPage/index.tsx

```diff
@@ -156,6 +182,15 @@ export class DashboardPage extends React.Component<DashboardPageProps, Dashboard
                       </Link>
                     )
                   }
+                  {
+                    dashboard.owners.length === 0 &&
+                    <AvatarLabel
+                      avatarColor='#CACAD9'
+                      avatarTextColor='#CACAD9'
+                      labelClass='text-placeholder'
+                      label={NO_OWNER_TEXT}
```

Comment:
Nit: I'm not the biggest fan of passing color codes directly to components. I know the Avatar component directly adds color styles to the elements. We could get around that by adding some wrapper class `gray-avatar` (?) and assigning an `!important` color in css to one of the child elements.

Reply:
Since we have the new `labelClass` if I add an `avatarClass` and set and apply it correctly it will work.